src/Controller/Api/ClinicController.php

```diff
@@ -15,27 +15,19 @@
 use Symfony\Component\Serializer\Serializer;
 use Symfony\Component\HttpFoundation\Response;
 
-class ApiClinicController extends AbstractController
+class ClinicController extends AbstractController
 {
-    public function index()
-    {
-        return new JsonResponse(['status' => 'ok']);
-    }
-
     public function list()
```

Comment:
let's keep `index()` is more semantical in terms on REST APIs 
```suggestion
    public function index()
```